samples/hello-android/settings.gradle

```diff
@@ -1 +1,15 @@
+pluginManagement {
+    repositories {
+        gradlePluginPortal()
+        maven { url "https://jcenter.bintray.com/" }
+    }
+    resolutionStrategy {
+        eachPlugin {
+            if(requested.id.id == "com.android.application") {
```

Comment:
:x: Space missing between `if` and the parenthesis.

Reply:
Fixed